### ISSUE TITLE
Windows Build Issue : QLogger.h Error

### ISSUE THREAD

**Issue by Frick-David** [francescmm/GitQlient#57]
I get this error when running build in QtCreator:
'''
fatal error C1083: Cannot open include file: 'QLogger.h': No such file or directory
'''
It causes an QLogger.h cannot open error for a couple files.
I noticed you have the QLogger Repo here: https://github.com/francescmm/QLogger

I am trying to build this on Windows with QtCreator. Could you describe to me how I can get QLogger included? Should I git clone it into the same folder as GitQlient then run the QtCreator build?

**Comment by Frick-David** [comment]
See attached error log: It is possible it is not QLogger that is the issue
`13:12:48: Running steps for project GitQlient...
13:12:48: Starting: "C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\qmake.exe" C:\Utility-Software\GitQlient\GitQlient.pro -spec win32-msvc "CONFIG+=debug" "CONFIG+=qml_debug"
Info: creating stash file C:\Utility-Software\build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug\.qmake.stash
Cannot read C:/Utility-Software/GitQlient/QLogger/QLogger.pri: No such file or directory
13:12:49: The process "C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\qmake.exe" exited normally.
13:12:49: Starting: "C:\Qt\Qt5.14.1\Tools\QtCreator\bin\jom.exe" -f C:/Utility-Software/build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug/Makefile qmake_all

jom 1.1.3 - empower your cores

13:12:49: The process "C:\Qt\Qt5.14.1\Tools\QtCreator\bin\jom.exe" exited normally.
13:12:49: Starting: "C:\Qt\Qt5.14.1\Tools\QtCreator\bin\jom.exe" 
	C:\Qt\Qt5.14.1\Tools\QtCreator\bin\jom.exe -f Makefile.Debug
	C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\uic.exe ..\GitQlient\aux_widgets\BranchDlg.ui -o ui_BranchDlg.h
	C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\uic.exe ..\GitQlient\aux_widgets\CreateRepoDlg.ui -o ui_CreateRepoDlg.h
	C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\uic.exe ..\GitQlient\branches\AddSubmoduleDlg.ui -o ui_AddSubmoduleDlg.h
	C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\uic.exe ..\GitQlient\branches\TagDlg.ui -o ui_TagDlg.h
	C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\uic.exe ..\GitQlient\commits\WorkInProgressWidget.ui -o ui_WorkInProgressWidget.h
	C:\Qt\Qt5.14.1\5.14.1\msvc2017_64\bin\uic.exe ..\GitQlient\config\GitConfigDlg.ui -o ui_GitConfigDlg.h
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\main.obj.24908.125.jom
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\ClickableFrame.obj.24908.125.jom
main.cpp
ClickableFrame.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\GitQlient.obj.24908.188.jom
GitQlient.cpp
..\GitQlient\big_widgets\GitQlient.cpp(15): fatal error C1083: Cannot open include file: 'QLogger.h': No such file or directory
..\GitQlient\main.cpp(6): fatal error C1083: Cannot open include file: 'QLogger.h': No such file or directory
jom: C:\Utility-Software\build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug\Makefile.Debug [debug\main.obj] Error 2
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\GitQlientRepo.obj.24908.188.jom
GitQlientRepo.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\GitQlientSettings.obj.24908.203.jom
GitQlientSettings.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\GitQlientStyles.obj.24908.203.jom
GitQlientStyles.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\HistoryWidget.obj.24908.219.jom
HistoryWidget.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\BranchContextMenu.obj.24908.219.jom
BranchContextMenu.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\ProgressDlg.obj.24908.141.jom
ProgressDlg.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\BlameWidget.obj.24908.172.jom
BlameWidget.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\Controls.obj.24908.172.jom
Controls.cpp
	cl -c -nologo -Zc:wchar_t -FS -Zc:rvalueCast -Zc:inline -Zc:strictStrings -Zc:throwingNew -Zc:referenceBinding -Zc:__cplusplus -Zi -MDd -std:c++17 -W3 -w34100 -w34189 -w44996 -w44456 -w44457 -w44458 -wd4577 -wd4467 -EHsc /Fddebug\GitQlient.vc.pdb -DUNICODE -D_UNICODE -DWIN32 -D_ENABLE_EXTENDED_ALIGNED_STORAGE -DWIN64 -DQT_DEPRECATED_WARNINGS -DVER=\"1.0.0\" -DDEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_CORE_LIB -DQT_QML_DEBUG -I..\GitQlient -I. -I..\GitQlient\aux_widgets -I..\GitQlient\big_widgets -I..\GitQlient\branches -I..\GitQlient\commits -I..\GitQlient\config -I..\GitQlient\diff -I..\GitQlient\git -I..\GitQlient\history -I..\GitQlient\QLogger -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtWidgets -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtGui -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtANGLE -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\include\QtCore -Idebug -I. -I/include -I..\..\Qt\Qt5.14.1\5.14.1\msvc2017_64\mkspecs\win32-msvc -Fodebug\ @C:\Users\james\AppData\Local\Temp\DiffWidget.obj.24908.172.jom
DiffWidget.cpp
..\GitQlient\big_widgets\DiffWidget.cpp(8): fatal error C1083: Cannot open include file: 'QLogger.h': No such file or directory
..\GitQlient\big_widgets\GitQlientRepo.cpp(9): fatal error C1083: Cannot open include file: 'QLogger.h': No such file or directory
jom: C:\Utility-Software\build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug\Makefile.Debug [debug\DiffWidget.obj] Error 2
jom: C:\Utility-Software\build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug\Makefile.Debug [debug\GitQlientRepo.obj] Error 2
..\GitQlient\big_widgets\HistoryWidget.cpp(14): fatal error C1083: Cannot open include file: 'QLogger.h': No such file or directory
jom: C:\Utility-Software\build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug\Makefile.Debug [debug\HistoryWidget.obj] Error 2
jom: C:\Utility-Software\build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug\Makefile.Debug [debug\GitQlient.obj] Error 2
jom: C:\Utility-Software\build-GitQlient-Desktop_x86_windows_msvc2019_pe_64bit-Debug\Makefile [debug] Error 2
13:12:52: The process "C:\Qt\Qt5.14.1\Tools\QtCreator\bin\jom.exe" exited with code 2.
Error while building/deploying project GitQlient (kit: Desktop (x86-windows-msvc2019-pe-64bit))
The kit Desktop (x86-windows-msvc2019-pe-64bit) has configuration issues which might be the root cause for this problem.
When executing step "Make"
13:12:52: Elapsed time: 00:04.`

**Comment by francescmm** [comment]
Hi @Frick-David,

For building GitQlient from source code I created a guide: https://github.com/francescmm/GitQlient/blob/master/docs/SETUP_BUILD.md. Take into account that if you want stability, you should work only on master. I'll bump the branch at every release.

For the v1.0.0 I fixed the issues for Windows on MSVC compiler and MinGW, so it should work for both.

If you still have issues could you past some information about the compiler you're using?

**Comment by Frick-David** [comment]
So I am trying to build this nice repo in QT Creator.  The issue could be two parts.
1) It seems QT Creator wants to use MSVC 2017 (no option for 2019) and gives a warning when using MSVC2019 to build. This could potentially be the issue. I am not entirely what to do about that because Windows does not seem to want to let me uninstall 2019. 

2) So, I went the build from source (from powershell route).  I added qmake to my PATH (although I think I still had to give it the specific location) and it gave an error about needing to specify a compiler (which more or less leads back to first part.

I think these lead QT creator to not want to build QLogger so then I get an error about missing QLogger but it seems it is more a compiler error (probably the mismatch between 2017 and 2019). Is there any way around this that comes to mind?

**Comment by francescmm** [comment]
I recommend you to go to the Qt forum: https://forum.qt.io/ and paste all the information you can: folder structure of Qt, config of the project configuration, etc.

I don't know how you installed Qt and Qt Creator. If it's done through the online installer it shouldn't be that much of a problem. Later, I'll try to upload an image of how my config looks like in case it helps you.

**Comment by Frick-David** [comment]
Okay sounds good! Looking forward to the picture.

**Comment by Frick-David** [comment]
@francescmm I got it to build. I ended up using MinGW to build it. It seems there is some issues with QT Creator wanting MSVC 2017 and Win 10 has MSVC 2019 by default. Even though they are compatible, Creator does not like it. Oh well, If there a way for me to put a shortcut on my desktop pointing to the build of it? I think right now, I need to keep building it for each use.

Best, hope all is well.
